app/jobs/tts_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# This jobs generates TTS audio attachments for all words with enabled TTS but without audio attachment.
+# It iterates automatically through all words and ends, when there are no more words to process.
+# There should be a cronjob running this job every 5 minutes and avoid duplicates, to make sure, that the jobs
+# is always running.
+class TtsJob < ApplicationJob
+  queue_as :default
+
+  # There is a rate limit for the TTS API. We shouldn't hit that limit even without pause, but to be
+  # absolute sure, we wait a second after each request.
+  PAUSE = 1.second
+
+  def perform
+    @logger = Logger.new(Rails.root.join("log", "tts.log"))
+    @logger.info "Starting TTS job"
+
+    loop do
+      word = find_next_word
+
+      unless word
+        @logger.info "Found no more words to process. Exiting."
+        return true
+      end
+
+      generate_audio word
+      sleep PAUSE
+    end
+  end
+
+  private def generate_audio(word)
+    @logger.info "Processing word #{word.id} (#{word.name})"
+
+    word.audio.attach(
+      io: TtsGenerator.call(word.name),
```

Comment:
@wintermeyer We always wanted to show articles for nouns. What about speech? Does the audio need to include the article for nouns as well?

Reply:
> @wintermeyer We always wanted to show articles for nouns. What about speech? Does the audio need to include the article for nouns as well?

Good thinking!

@cabotine Do we want "das Auto" or "Auto" as a voice prompt?

Reply:
Yes, please include the articles! And thank you for thinking about it. Only edge case might be something like "der/die/das Joghurt". https://wort.schule/joghurt

Reply:
> Yes, please include the articles! And thank you for thinking about it. Only edge case might be something like "der/die/das Joghurt". https://wort.schule/joghurt

@phortx Please add the article for Nomen. In case there are multiple articles: Use the first one.

Reply:
Done